package.json

```diff
@@ -174,22 +174,12 @@
     "react-native-web": "^0.19.12",
     "react-native-web-sound": "^0.1.3",
     "react-native-webview": "13.8.6",
-    "react-pdf": "9.1.0",
```

Comment:
These changes work well, but why are we removing react-pdf from package.json?

Reply:
It might result in different versions of the same package being installed if the versions get out of sync (see https://github.com/Expensify/App/pull/41238).

There are a few other packages not included in `packages.json`, including `pdfjs-dist`. Ideally we should re-enable the [`import/no-extraneous-dependencies`](https://github.com/Expensify/App/blob/71db3f19691069182447b6b1b9fff62ec93be86c/.eslintrc.js#L184) rule and add these packages to `peerDependencies` as we do in [`react-fast-pdf`](https://github.com/Expensify/react-fast-pdf/blob/5079336fcdf5d93df2ace257bd199414f313b9af/package.json#L35-L40)

Reply:
okay okay, I don't have a strong opinion about removing it from the package.json. @thienlnam , are you alright with us removing it from the package.json?

Reply:
No opinion here either - should be okay

Reply:
Okay, we can proceed with the merge. The PR has tested well.